src/main/scala/cromwell/engine/SymbolStore.scala

```diff
@@ -26,9 +26,46 @@ class SymbolStore(binding: WdlBinding, inputs: Map[FullyQualifiedName, WdlValue]
     assignSymbolStoreEntry(fullyQualifiedName, value, input = true)
   }
 
+  binding.workflows foreach {workflow =>
+    workflow.calls foreach {call =>
+      call.inputMappings foreach {
+        case (k, v) => assignSymbolStoreEntry(s"${call.fullyQualifiedName}.$k", v, input = true)
+      }
+    }
+  }
+
   def locallyQualifiedInputs(call: Call): Map[String, WdlValue] = {
+
+    def lookup(s: String): WdlValue = {
+      var workflow = call.parent.getOrElse {
+        throw new WdlExpressionException("Expecting 'call' to have a 'workflow' parent")
+      }.asInstanceOf[Workflow]
```

Comment:
It might be nice to reverse this, something like:

    call.parent.map {_.asInstanceOf[Workflow]} getOrElse {blah blah blah}